src/Microsoft.Azure.SignalR.SignalRService/ServiceManager.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.SignalR.SignalRService
+{
+    public class ServiceManager
+    {
+        public ServiceHubContext CreateHubContextAsync(string hubName)
```

Comment:
Should return `Task<ServiceHubContext>`?

Reply:
Yes